app/operations/workflows/retire_subjects.rb

```diff
@@ -1,6 +1,9 @@
 module Workflows
   class RetireSubjects < Operation
```

Comment:
This operation should (for now) rewrite incoming "blank" reasons to "nothing_here". Nero sends that, so we'd break the API if we don't. It's not a public API method as far as I can tell from the docs, so this is just so we decouple the Nero update from this one.